x/dualstaking/keeper/hooks.go

```diff
@@ -133,7 +133,7 @@ func (h Hooks) BeforeDelegationRemoved(ctx sdk.Context, delAddr sdk.AccAddress,
 	if err != nil {
 		return nil
 	}
-	amount := validator.TokensFromShares(delegation.Shares).TruncateInt()
+	amount := validator.TokensFromSharesRoundUp(delegation.Shares).TruncateInt()
 	err = h.k.UnbondUniformProviders(ctx, delAddr.String(), sdk.NewCoin(epochstoragetypes.TokenDenom, amount))
```

Comment:
if unbond uniform fails, do we really want to fail the hook? didn't we say we will allow some cases? maybe let's allow failing unbond uniform in the special case it's the last delegation, i think this should cover our edge case

Reply:
unbonduniform will fail only if something is wrong with the delegations,
so if we want to unbond more than we have it wont sent an error